fastai/vision/augment.py

```diff
@@ -596,6 +592,16 @@ def __init__(self,min_zoom=1., max_zoom=1.1, p=0.5, draw=None, draw_x=None, draw
         aff_fs = partial(zoom_mat, min_zoom=min_zoom, max_zoom=max_zoom, p=p, draw=draw, draw_x=draw_x, draw_y=draw_y, batch=batch)
         super().__init__(aff_fs, size=size, mode=mode, pad_mode=pad_mode, align_corners=align_corners)
 
+# Internal Cell
+def _linalg_solve(A,B):
+    return torch.linalg.solve(A,B)
+
+def _solve(A,B):
+    return torch.solve(B,A)[0]
+
+if torch.__version__ >= '1.9': solve = _linalg_solve
```

Comment:
@jph00   won't this string compare for version fail? ie '1.10.0' >= '1.9' is False